city_scrapers/spiders/wayne_state_university.py

```diff
@@ -0,0 +1,95 @@
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+import re
+
+
+class WayneStateUniversitySpider(CityScrapersSpider):
+    name = "wayne_state_university"
+    agency = "Wayne State University"
+    timezone = "America/Chicago"
+    allowed_domains = ["bog.wayne.edu"]
+    start_urls = ["http://bog.wayne.edu/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        
+        #one large all encompassing meeting
+        meeting_dates = response.xpath('//*[@id="content"]/div[1]/div/h2').re('((?:January|Feburary|March|April|May|June|July|August|October|September|November|December)\s+\d{1,2},\s+\d{4})')
+        for item in meeting_dates:
+            meeting = Meeting(
```

Comment:
For each date you'll want to yield 3 meetings, one for each of the separate categories (since they have distinct start times). You can parse those in the beginning instead of re-running it each time in `_parse_start`.

This could look like a method that does something like this:

```python
def _parse_meeting_types(self, response):
    """Parse meeting titles and start times. Returns a list of tuples including title and start time"""
    meeting_details = []
    for item in response.css("#content .border-t li"):
        title, start_time = self._parse_title_start_time(item) # Placeholder function
        meeting_details.append(title, start_time)
    return meeting_details
```

Then you could iterate first through dates, and then inside that loop create another loop that iterates through each of the meeting types